extensions/amp-story-interactive/0.1/amp-story-interactive-img-poll.js

```diff
@@ -37,7 +61,53 @@ export class AmpStoryInteractiveImgPoll extends AmpStoryInteractive {
 
   /** @override */
   buildComponent() {
-    this.rootEl_ = htmlFor(this.element)`<p>Image poll component</p>`;
+    this.rootEl_ = buildImgTemplate(this.element);
+    this.attachContent_(this.rootEl_);
     return this.rootEl_;
   }
+
+  /**
+   * Finds the prompt and options content
+   * and adds it to the poll element.
+   *
+   * @private
+   * @param {Element} root
+   */
+  attachContent_(root) {
+    this.attachPrompt_(root);
+
+    const optionContainer = this.rootEl_.querySelector(
+      '.i-amphtml-story-interactive-img-option-container'
+    );
+    this.options_.forEach((option, index) =>
+      optionContainer.appendChild(this.configureOption_(option, index))
+    );
+  }
+
+  /**
+   * Creates and returns an option container with option content,
+   * adds styling and answer choices.
+   *
+   * @param {!./amp-story-interactive-abstract.OptionConfigType} option
+   * @return {!Element}
+   * @private
+   */
+  configureOption_(option) {
+    const convertedOption = buildOptionTemplate(this.element);
+    convertedOption.optionIndex_ = option['optionIndex'];
+
+    // Extract and structure the option information
+    // TODO: Rewrite image URL (https://github.com/ampproject/amphtml/pull/35043#discussion_r660874389)
```

Comment:
Can you file a ticket instead and track it in the appropriate GitHub project? This is very important

Reply:
I'll submit tickets for these things on the interactives project